guac-SBOM-ingestion.md

```diff
@@ -0,0 +1,88 @@
+## SBOM Ingestion in GUAC
+
+### Overview
+
+Software Bill of Materials (SBOM) ingestion is essential in GUAC (Graph for Understanding Artifact Composition) to help track and analyze dependencies, vulnerabilities, and software supply chain metadata. This documentation explains how to ingest SBOMs in GUAC, covering:
+- **Supported formats and limitations**
+- **Ingestion methods**
+- **Daemon-mode ingestion**
+- **Manual vs. automatic ingestion** 
+
+---
+
+### Supported SBOM Formats
+
+GUAC supports a variety of SBOM formats, making it compatible with several tools and standards:
+- **SPDX**: A widely used open standard for software package metadata.
+- **CycloneDX**: An SBOM specification built for security use cases.
+- **Syft**: A tool generating SBOMs directly from container images and filesystems.
```

Comment:
I don't believe we support Syft's native format, just the SPDX and CycloneDX output that Syft can produce. @pxp928 or someone can fact check me.